File: LeetCode/Easy/Maximum Number of Words Found in Sentences/2114.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxCnt = 0;

        for (int i = 0; i < sentences.length; i++) {
            int cnt = 1;

            for (int j = 0; j < sentences[i].length(); j++) {
                if (sentences[i].charAt(j) == ' ') {
                    cnt++;
                }
            }

            if (cnt > maxCnt) {
                maxCnt = cnt;
            }
        }

        return maxCnt;
    }
}

File: LeetCode/Easy/Maximum Number of Words Found in Sentences/2114.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxCnt = 0;

    for (let i = 0; i < sentences.length; i++) {
        let cnt = 1;

        for (let j = 0; j < sentences[i].length; j++) {
            if (sentences[i].charAt(j) === ' ') {
                cnt++;
            }
        }

        if (cnt > maxCnt) {
            maxCnt = cnt;
        }
    }

    return maxCnt;
};
